lua/nvim-treesitter/query.lua

```diff
@@ -12,7 +12,11 @@ end
 function M.get_query(ft, query_name)
   local query_files = api.nvim_get_runtime_file(string.format('queries/%s/%s.scm', ft, query_name), false)
   if #query_files > 0 then
-    return ts.parse_query(ft, read_query_file(query_files[1]))
+    local query_str = read_query_file(query_files[1])
+    local query =  ts.parse_query(ft, query_str)
+    if query then return query end
+
+    print("Error :", query_str)
```

Comment:
this error is not very explicit, did you do that only for debugging purposes ?

Reply:
Oh yeah, I forgot that sorry